lookout/style/format/analyzer.py

```diff
@@ -107,7 +108,9 @@ def group_line_nodes(y, y_pred, vnodes_y, rule_winners):
                     log.warning("Failed to parse %s", file.path)
                     continue
                 X, y, vnodes_y, vnodes = res
-                y_pred, rule_winners = rules.predict(X, vnodes_y, vnodes, fe)
+                client = BblfshClient(self.config["bblfsh_address"])
+                y_pred, rule_winners = rules.predict(X, y, vnodes_y, vnodes, {file.path: file},
+                                                     fe, client)
```

Comment:
Our AST check is clearly abstract from applying the list of rules, compared to "quote hacks". It works with any prediction, whether the origin is rules, deep learning or smth else.

Thus It should be implemented as a separate filter.